### ISSUE TITLE
email notifications for likes

### ISSUE THREAD

**Issue by paulbutenko** [cat-in-136/redmine_hearts#46]
Hi 
this plugin is really great for socialization, thank you
I have mentioned that not that much our redmine's users are checking hearts page in order to see likes.

wondering if it's possible to add email notification for issues and comments likes?
it would be great if users could configure on their profile pages if they would like to get such notifications.

what do you think?

**Comment by pkoevesdi** [comment]
I agree, but want to stress to make this pure optional. We use hearts kind of as a reading confirmation, for which a confirmation is not wished.

**Comment by cat-in-136** [comment]
I don't want it, but I think there are people who want. 
I agree with @pkoevesdi from design perspective, and since the default redmine email notification is carefully implemented for same reasons, it should be an opt-in feature.
I probably won't take the time to implement it, but if a PR provided, I will accept it.
